### ISSUE TITLE
Allow embedded links in multiformatMessage.text

### ISSUE THREAD

**Issue by ghost** [oasis-tcs/sarif-spec#471]
§3.11.6 says that `message.text` can contain embedded links. But he spec does not say that `multiformatMessageString.text` (§3.12.3) can contain embedded links. It should say that.

@michaelcfanning FYI

* * * *

**FINAL PROPOSAL:**

**§3.12 multiformatMessageString object
§3.12.3 text property**

A `multiformatMessageString` object **SHALL** contain a property named `text` whose value is a non-empty string containing a plain text representation of the message including any links.

> NOTE: This property is required to ensure that the message is viewable even in contexts that do not support the rendering of formatted text.

**Comment by schlaman-ms** [comment]
Document location for issue:

lgolding:
"The spec does not say that multiformatMessageString.text (§3.12.3) can contain embedded links. It should say that."

Consider updating the following property to include mention of "can contain embedded links..."

§3.12.3 text property

**Comment by stacywray** [comment]
**§3.12 multiformatMessageString object
§3.12.3 text property**

A `multiformatMessageString` object **SHALL** contain a property named `text` whose value is a non-empty string containing a plain text representation of the message and **MAY** contain embedded links.

> NOTE: This property is required to ensure that the message is viewable even in contexts that do not support the rendering of formatted text.

**Comment by sthagen** [comment]
I feel more comfortable with:

```
\[...] a plain text representation of the message including any links.
```

As an implementer, I find the `MAY` in a sentence that contains a `SHALL` irritating. Especially when a consumer can always choose to ignore or analyze text values ​​at their own discretion.

**Comment by sthagen** [comment]
Implemented per #636.
